openprescribing/dmd/management/commands/fetch_dmd.py

```diff
@@ -32,6 +32,10 @@ def handle(self, *args, **kwargs):
             "commit": "LOG+IN",
         }
         rsp = session.post(login_url, params)
+        if "The email address or password is invalid" in rsp.text:
+            raise RuntimeError(
+                f"TRUD login failure for username: {settings.TRUD_USERNAME!r}"
+            )
```

Comment:
This could also post to the data team slack channel with `notify_slack(message, is_error=True)`